lib/pwinty/order.rb

```diff
@@ -17,6 +17,7 @@ class Order < Pwinty::Base
         attribute :stateOrCounty, Types::String.optional
         attribute :status, Types::String
         attribute :payment, Types::String
+        attribute :packingSlipUrl, Types::String.optional
```

Comment:
```suggestion
        attribute? :packingSlipUrl, Types::String.optional
```

Reply:
This change will make the attribute *omittable*, as well as the value being optional

Reply:
If you commit this suggestion I'll be happy to merge this into master and bump the version to v3.0.4

Otherwise, I'll push a similar change next week - but I'd rather see you get the credit in the contributors list.